elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -139,4 +139,28 @@ default void flush() {
                                                           Optional<Sorting> sorting, Optional<Pagination> pagination) {
         return collection;
     }
+
+    default <T> Object getRelation(
+            Object entity,
+            RelationshipType relationshipType,
+            String relationName,
+            Class<T> relationClass,
+            EntityDictionary dictionary,
+            Set<Predicate> filters
+    ) {
+        return PersistentResource.getValue(entity, relationName, dictionary);
```

Comment:
Logically, something feels wrong calling in from `DataStoreTransaction` to `PersistentResource`. Maybe we should find a common place to put the reflection code that runs `getValue()`?